packages/app/src/templates/components/templates/templates.component.test.tsx

```diff
@@ -163,4 +177,68 @@ describe("`Templates` component", () => {
       expect(getByText("Some error")).toBeInTheDocument()
     })
   })
+  describe("`deleteTemplate` functionality", () => {
+    test("show deleting success modal, when deletingState is Success", async () => {
+      const { getByText } = await render({
+        templates: [templateMock],
+        createTemplate: createTemplateMock,
+        deleteTemplates: deleteTemplatesMock,
+        loading: false,
+        error: null,
+        deletingState: TemplateDeletingState.Success,
+        hideDeleteModal: hideDeleteModalMock,
+      })
+
+      expect(
+        getByText("[value] module.templates.templateDelete")
+      ).toBeInTheDocument()
+    })
+    test("show deleting success modal, when deletingState is Fail", async () => {
+      const { getByText } = await render({
+        templates: [templateMock],
+        createTemplate: createTemplateMock,
+        deleteTemplates: deleteTemplatesMock,
+        loading: false,
+        error: null,
+        deletingState: TemplateDeletingState.Fail,
+        hideDeleteModal: hideDeleteModalMock,
+      })
+
+      expect(
+        getByText("[value] module.templates.deleteModalErrorSubtitle")
+      ).toBeInTheDocument()
+    })
+    test("show deleting loader, when deletingState is Deleting", async () => {
```

Comment:
The description says that we show `deleting loader` while in the test we check if `deletingModalTitle` is shown. Is it correct?

Reply:
Test fixed in [7c222ee](https://github.com/mudita/mudita-center/pull/764/commits/7c222eed5a12a682f3d9b7d9ec91c06d8a7edc54)